src/main/java/redis/clients/jedis/exceptions/JedisConfigException.java

```diff
@@ -0,0 +1,21 @@
+package redis.clients.jedis.exceptions;
+
+/**
+ * A config error.
+ */
+public class JedisConfigException extends JedisException {
```

Comment:
Remove this exception. A simple `JedisException` with proper exception message is enough to serve the purpose ATM.

Reply:
All right. I'll provided this correction. Thanks.